docs/components/buttons/index.html

```diff
@@ -82,33 +106,4 @@ <h3>Button Group</h3>
       </div>
     </div>
   </section>
-
-  <section data-visreg="button-states">
```

Comment:
Given that I removed the button states from this page, I added a "See Also" link to the test page, if consumers wish to see them.